src/main/java/hudson/security/LDAPSecurityRealm.java

```diff
@@ -903,10 +895,10 @@ public GroupDetails loadGroupByGroupname(String groupname, boolean fetchMembers)
                     return groupDetails;
                 }
             // Make sure we don't throw BadCredentialsException. Catch logic matches LDAPUserDetailsService#loadUserByUsername.
-            } catch (DataAccessException e) {
```

Comment:
Not really sure what the Spring Security implementation is going to throw instead of this, so just guessing.